packages/ckeditor5-dev-tests/tests/utils/manual-tests/compilehtmlfiles.js

```diff
@@ -14,6 +14,13 @@ const chokidar = require( 'chokidar' );
 const fakeDirname = path.dirname( require.resolve( '../../../lib/utils/manual-tests/compilehtmlfiles' ) );
 
 describe( 'compileHtmlFiles', () => {
+	// This test is disabled on Windows
+	// See https://github.com/ckeditor/ckeditor5/issues/10538
+	if ( process.platform === 'win32' ) {
+		console.log( 'This test is disabled on Windows' );
+		return;
+	}
```

Comment:
It would be good to explain why the test is disabled.

```suggestion
	// Temporary these tests are disabled on Windows.
	// See https://github.com/ckeditor/ckeditor5/issues/10538.
	if ( process.platform === 'win32' ) {
		console.log( 'The "compileHtmlFiles()" tests do not work on Windows environments.' );
		console.log( 'See: https://github.com/ckeditor/ckeditor5/issues/10538 for more details.' );

		return;
	}
```